### PR TITLE
CCIA HRA makeover

### PR DESCRIPTION
**Pull request by nauticall**
Overhauls the look of HRAs from CCIA; rather than using the stock SCC liaison outfit, they now come with a proper coat and tie. Subjectively, this makes them look twice as intimidating and five times more likely for people to take them seriously.

Additionally, this PR gives HRA agents a baton, an energy pistol (and holster), and a laptop. Gotta be prepared.

Pics below.

imageimageimageimage

_Loadout._
image

EDIT 12/22: Found a way to give pistol holsters and ties both to the agents, thanks to Camo.

### CODE REVIEW

**Comment by nauticall** [comment]
!review

File: config/example/admins.txt [hunk 1]
```diff
@@ -5,4 +5,4 @@
 # Ranks can be anything defined in admin_ranks.json          ~Carn   #
 ######################################################################
 
-# not_a_user - Head Admin/Dev
\ No newline at end of file
+# Nauticall - Head Admin/Dev
```
Comment (Generalcamo) [inline config/example/admins.txt:8]: Seems this was accidentally left in?

File: code/modules/clothing/factions/scc.dm [hunk 2]
```diff
@@ -35,6 +41,36 @@
 /obj/item/clothing/suit/storage/toggle/armor/vest/scc/toggle_open()
 	return
 
+/obj/item/clothing/suit/storage/toggle/armor/ccia
```
Comment (Generalcamo) [inline code/modules/clothing/factions/scc.dm:44]: Wouldn't it be better to make this into a subset of `suit/storage`? You're not using the functionality of `toggle` at all, and indeed you remove it below.

**Review by Generalcamo — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Comment by alsoandanswer** [comment]
wouldn't it be better for the holster to come auto-attached?

**Comment by nauticall** [comment]
> wouldn't it be better for the holster to come auto-attached?

Fixed in last commit https://github.com/Aurorastation/Aurora.3/pull/18029/commits/61e9fea8891921a6bb916a2be38d61b5834274f9

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**